packages/nodes-base/nodes/Slack/ChannelDescription.ts

```diff
@@ -20,86 +20,103 @@ export const channelOperations: INodeProperties[] = [
 				name: 'Archive',
 				value: 'archive',
 				description: 'Archives a conversation',
+				action: 'Archive a channel',
 			},
 			{
 				name: 'Close',
 				value: 'close',
 				description: 'Closes a direct message or multi-person direct message',
+				action: 'Close a channel',
 			},
 			{
 				name: 'Create',
 				value: 'create',
 				description: 'Initiates a public or private channel-based conversation',
+				action: 'Create a channel',
 			},
 			{
 				name: 'Get',
 				value: 'get',
 				description: 'Get information about a channel',
+				action: 'Get a channel',
 			},
 			{
 				name: 'Get All',
 				value: 'getAll',
 				description: 'Get all channels in a Slack team',
+				action: 'Get all channels',
 			},
 			{
 				name: 'History',
 				value: 'history',
 				description: 'Get a conversation\'s history of messages and events',
+				action: 'History a channel',
```

Comment:
History of a channel?

Reply:
Changed to `Get the history of a channel`